### ISSUE TITLE
DDL downloads cannot be found 

### ISSUE THREAD

**Issue by misleadingrhino** [evilhero/mylar#2263]
**Describe the bug**
Downloads are being sent to the cache folder and tagged correctly, but the post-processor can't seem to find them when they're done. The tagged file is in the folder. One of the lines near the end makes it look like the post-processor is looking for the original filename (cbr) instead of the recompressed cbz. 

**Screenshots/Logs**
```
24-Apr-2019 13:16:03 - INFO    :: mylar.ddl_downloader.3084 : DDL-QUEUE : Skyward 012 (2019) (Digital) (Zone-Empire).cbr successfully downloaded - now initiating post-processing.
24-Apr-2019 13:16:07 - INFO    :: mylar.postprocess_main.3111 : POST-PROCESS-QUEUE : Now loading from post-processing queue: {'issueid': u'706868', 'apicall': True, 'nzb_name': u'Skyward 012 (2019) (Digital) (Zone-Empire).cbr', 'nzb_folder': u'/config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr', 'failed': False, 'ddl': True, 'comicid': u'109973'}
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.358 : Thread-50 : [POST-PROCESSING] nzb name: Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.359 : Thread-50 : [POST-PROCESSING] nzb folder: /config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.386 : Thread-50 : [POST-PROCESSING] Now performing post-processing of Skyward 012 (2019) (Digital) (Zone-Empire).cbr sent from DDL
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.419 : Thread-50 : [POST-PROCESSING] Now post-processing directly against ComicID: 109973 / IssueID: 706868
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.287 : Thread-50 : Scanner group tag discovered: Zone-Empire
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.394 : Thread-50 : NEWLY SPLIT REORGD: [u'Skyward', u'012', u'(2019)', u'(Digital)']
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.429 : Thread-50 : checking date : 2019
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.432 : Thread-50 : date: 2019
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.759 : Thread-50 : datecheck: [{'date': u'2019', 'position': 2, 'mod_position': 12}]
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.765 : Thread-50 : sctd: 2019
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.774 : Thread-50 : year verified as : 2019
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.780 : Thread-50 : year verified as: 2019
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.810 : Thread-50 : highest_series_position: 2
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.815 : Thread-50 : possible_issuenumbers: [{'position': 1, 'validcountchk': False, 'number': u'012', 'mod_position': 8}]
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.858 : Thread-50 : issue number :012
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.887 : Thread-50 : issue_position: 1
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.904 : Thread-50 : issue verified as : 012
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.1031 : Thread-50 : sf_highest_series_pos: [u'Skyward']
24-Apr-2019 13:16:07 - DEBUG   :: mylar.parseit.1076 : Thread-50 : series title possibly: Skyward
24-Apr-2019 13:16:07 - INFO    :: mylar.Process.541 : Thread-50 : Now checking: Skyward [109973]
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.726 : Thread-50 : [POST-PROCESSING][ISSUE-VERIFY] Now checking against Skyward - 109973
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.801 : Thread-50 : [POST-PROCESSING][ISSUE-VERIFY][Lone Volume MATCH] Volume label of None indicates only volume for this series on your watchlist.
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.809 : Thread-50 : [POST-PROCESSING][ISSUE-VERIFY][Issue Year MATCH] Issue Year of 2019 is a match to the year found in the filename of : 2019
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.833 : Thread-50 : [POST-PROCESSING][CLOCATION] /config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.876 : Thread-50 : [POST-PROCESSING][DEFINITIVE-NAME MATCH] Definitive name match exactly to : Skyward [109973]
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.884 : Thread-50 : [POST-PROCESSING][SUCCESSFUL MATCH: Skyward-109973] Match verified for Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:07 - DEBUG   :: mylar.Process.1318 : Thread-50 : [POST-PROCESSING] There are 1 files found that match on your watchlist, 0 files are considered one-off's, and 0 files do not match anything
24-Apr-2019 13:16:10 - INFO    :: mylar.torrents.162 : Thread-15 : Now retrieving feed from 32P
24-Apr-2019 13:16:11 - INFO    :: mylar.torrents.371 : Thread-15 : [32P] there were 0 results from your Personal Notification Feed : Spider-Woman
24-Apr-2019 13:16:11 - DEBUG   :: mylar.rssdbupdate.642 : Thread-15 : Completed adding new data to RSS DB. Next add in 20 minutes
24-Apr-2019 13:16:12 - INFO    :: mylar.postprocess_main.3125 : POST-PROCESS-QUEUE : Another item is post-processing still...
24-Apr-2019 13:16:18 - INFO    :: mylar.duplicate_filecheck.2140 : Thread-50 : [DUPECHECK] Duplicate check for /config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:18 - INFO    :: mylar.duplicate_filecheck.2272 : Thread-50 : [DUPECHECK] Duplication detection returned no hits. This is not a duplicate of anything that I have scanned in as of yet.
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2068 : Thread-50 : [POST-PROCESSING] Was not snatched as a torrent. Using manual post-processing.
24-Apr-2019 13:16:19 - INFO    :: mylar.Process_next.2075 : Thread-50 : [POST-PROCESSING]  [1/1] Starting Post-Processing for Skyward issue: 12
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2078 : Thread-50 : [POST-PROCESSING] issueid: 706868
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2079 : Thread-50 : [POST-PROCESSING] issuenumOG: 12
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2162 : Thread-50 : [POST-PROCESSING] Zero Suppression set to : 00x
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2202 : Thread-50 : issue detected greater than 10, but less than 100
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2217 : Thread-50 : [POST-PROCESSING] Zero level supplement set to 00x. Issue will be set as : 012
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2234 : Thread-50 : [POST-PROCESSING] Pretty Comic Issue is : 012
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2237 : Thread-50 : [POST-PROCESSING] Issue Year : 2019
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2244 : Thread-50 : [POST-PROCESSING] Publisher: Image
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2248 : Thread-50 : [POST-PROCESSING] Series: Skyward
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2256 : Thread-50 : [POST-PROCESSING] Year: 2018
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2259 : Thread-50 : [POST-PROCESSING] Comic Location: /comics/Skyward (2018)
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2262 : Thread-50 : [POST-PROCESSING] Comic Version: None
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2271 : Thread-50 : [POST-PROCESSING] No version # found for series, removing from filename
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2272 : Thread-50 : [POST-PROCESSING] New format is now: $Series $VolumeY $Issue ($Year)
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2280 : Thread-50 : [POST-PROCESSING] Not an annual - removing from filename parameters
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2281 : Thread-50 : [POST-PROCESSING] New format: $Series $VolumeY $Issue ($Year)
24-Apr-2019 13:16:19 - DEBUG   :: mylar.Process_next.2348 : Thread-50 : [POST-PROCESSING] Metatagging enabled - proceeding...
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.26 : Thread-50 : [META-TAGGER] dirName:/config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.31 : Thread-50 : ComicTagger Path location for internal comictagger.py set to : /app/mylar/comictagger.py
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.35 : Thread-50 : [META-TAGGER] Filename is : /config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.50 : Thread-50 : Filepath: /config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.51 : Thread-50 : Filename: Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.53 : Thread-50 : New_Folder: /config/mylar/cache/mylar_L39xSH
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.55 : Thread-50 : New_Filepath: /config/mylar/cache/mylar_L39xSH/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.73 : Thread-50 : [META-TAGGER] Paths / Locations:
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.74 : Thread-50 : [META-TAGGER] scriptname : Mylar.py
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.75 : Thread-50 : [META-TAGGER] downloadpath : /config/mylar/cache/Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.76 : Thread-50 : [META-TAGGER] sabnzbdscriptpath : /app/mylar
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.77 : Thread-50 : [META-TAGGER] comicpath : /config/mylar/cache/mylar_L39xSH
24-Apr-2019 13:16:19 - DEBUG   :: mylar.run.78 : Thread-50 : [META-TAGGER] Running the ComicTagger Add-on for Mylar
24-Apr-2019 13:16:20 - DEBUG   :: mylar.run.119 : Thread-50 : [META-TAGGER] /usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version! being used - using personal ComicVine API key supplied via mylar.
24-Apr-2019 13:16:20 - DEBUG   :: mylar.run.131 : Thread-50 : [META-TAGGER] CR Tagging enabled.
24-Apr-2019 13:16:20 - DEBUG   :: mylar.run.136 : Thread-50 : [META-TAGGER] CBL Tagging enabled.
24-Apr-2019 13:16:20 - DEBUG   :: mylar.run.190 : Thread-50 : [META-TAGGER] Enabling ComicTagger script: ['/usr/bin/python', '/app/mylar/comictagger.py'] with options: ['-e', u'/config/mylar/cache/mylar_L39xSH/Skyward 012 (2019) (Digital) (Zone-Empire).cbr']
24-Apr-2019 13:16:20 - DEBUG   :: mylar.run.198 : Thread-50 : [META-TAGGER] Executing command: ['/usr/bin/python', '/app/mylar/comictagger.py', '-e', u'/config/mylar/cache/mylar_L39xSH/Skyward 012 (2019) (Digital) (Zone-Empire).cbr']
24-Apr-2019 13:16:20 - DEBUG   :: mylar.run.199 : Thread-50 : [META-TAGGER] Absolute path to script: /usr/bin/python
24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.207 : Thread-50 : [META-TAGGER][COMIC-TAGGER] : /usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Archive exported successfully to: Skyward 012 (2019) (Digital) (Zone-Empire).cbz

24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.229 : Thread-50 : [META-TAGGER]Trying utf-8 conversion.
24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.233 : Thread-50 : [META-TAGGER]Trying latin-1 conversion.
24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.237 : Thread-50 : [META-TAGGER][COMIC-TAGGER][CBR-TO-CBZ] New filename: /usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Skyward 012 (2019) (Digital) (Zone-Empire).cbz
24-Apr-2019 13:16:22 - INFO    :: mylar.run.184 : Thread-50 : [META-TAGGER] ComicRack tagging meta-tagging processing started.
24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.194 : Thread-50 : [META-TAGGER] Enabling ComicTagger script: ['/usr/bin/python', '/app/mylar/comictagger.py'] with options: ['-s', '-m', 'volume=2018', '--cv-api-key', u'REDACTED', '-o', '--id', u'706868', '--type', 'cr', "/usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version!\n  RequestsDependencyWarning)\nSkyward 012 (2019) (Digital) (Zone-Empire).cbz"]
24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.198 : Thread-50 : [META-TAGGER] Executing command: ['/usr/bin/python', '/app/mylar/comictagger.py', '-s', '-m', 'volume=2018', '--cv-api-key', u'REDACTED', '-o', '--id', u'706868', '--type', 'cr', "/usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version!\n  RequestsDependencyWarning)\nSkyward 012 (2019) (Digital) (Zone-Empire).cbz"]
24-Apr-2019 13:16:22 - DEBUG   :: mylar.run.199 : Thread-50 : [META-TAGGER] Absolute path to script: /usr/bin/python
24-Apr-2019 13:16:22 - INFO    :: mylar.torrents.162 : Thread-15 : Now retrieving feed from 32P
24-Apr-2019 13:16:22 - INFO    :: mylar.torrents.371 : Thread-15 : [32P] there were 0 results from your Personal Notification Feed : Doctor+Strange
24-Apr-2019 13:16:22 - DEBUG   :: mylar.rssdbupdate.642 : Thread-15 : Completed adding new data to RSS DB. Next add in 20 minutes
24-Apr-2019 13:16:23 - DEBUG   :: mylar.run.254 : Thread-50 : [META-TAGGER] Output: /usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Cannot find /usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Skyward 012 (2019) (Digital) (Zone-Empire).cbz

24-Apr-2019 13:16:23 - WARNING :: mylar.run.255 : Thread-50 : [META-TAGGER][COMIC-TAGGER] Unable to locate file: Skyward 012 (2019) (Digital) (Zone-Empire).cbr
24-Apr-2019 13:16:23 - ERROR   :: mylar.Process_next.2394 : Thread-50 : [POST-PROCESSING] The file cannot be found in the location provided [Skyward 012 (2019) (Digital) (Zone-Empire).cbr]. Please verify it exists, and re-run if necessary. Aborting
24-Apr-2019 13:16:23 - INFO    :: mylar.Process.1666 : Thread-50 : [POST-PROCESSING] post-processing of pack completed for 1 issues [FAILED: 1]
```

**Environment (please complete the following information):**
 - OS: Linux Docker
 - Branch / commit: Dev/ee3f1cfa0cf3d14b3ffc6dc27e22f66a40dfe595

**Additional information**
[This container](https://github.com/misleadingrhino/docker-mylar) is based on the linuxserver dockerfile, just changed so that it pulls development instead of master.

**Comment by evilhero** [comment]
You have the same traceback repeated several times in log - basically you need to update your urllib3 and/or charset python modules (or make sure they're at the required compatible level).

**Comment by misleadingrhino** [comment]
That was it. I didn't think that was related, but it was. Thanks!

**Comment by blindguy** [comment]
> You have the same traceback repeated several times in log - basically you need to update your urllib3 and/or charset python modules (or make sure they're at the required compatible level).

I just set up a new mylar and am having what appear to the same problems with the ddl.

I tried the google machine but not sure I can find the proper docs to  update your urllib3 and/or charset python modules (or make sure they're at the required compatible level).

I think I updated the urllib3 not sure about the python.

Now that I updated the urllib3 how can I trigger a reprocess of the files in my cache folder?



Running on debian - fresh pull.

Configuration Options
Mylar Version: development
-- git build 7696edbc8b6fd893550965a0ea892143e53e23bd.
Python Version : 2.7.13
Language : en_US.UTF-8

MYLAR PROGRAM OPTIONS
Mylar Data Directory : /opt/mylar
Mylar Program Directory : /opt/mylar
Mylar Cache Directory : /opt/mylar/cache
Mylar Config File :/opt/mylar/config.ini
